package.json

```diff
@@ -95,8 +95,8 @@
         "prop-types": "^15.7.2",
         "query-selector-shadow-dom": "^1.0.0",
         "rc-trigger": "^5.2.5",
-        "react": "^16.13.1",
-        "react-dom": "^16.12.0",
+        "react": "^16.14.0",
```

Comment:
We have already de facto been on 16.14.0 – https://github.com/PostHog/posthog/blob/a406f00aa02e3ae0a2a2db33cb330baf75ea3921/yarn.lock#L15578-L15579